analytics/models/intermediate/int_iMER.sql

```diff
@@ -0,0 +1,19 @@
+SELECT
+    'orientation'                                                                       AS kind,
+    orientation_creation_date                                                           AS date,
+    orientation_prescriber_id                                                           AS user_id,
+    COALESCE(orientation_di_service_id IS NOT NULL, FALSE)                              AS is_di_service,
```

Comment:
thought: j'ai l'impression qu'on peut se passer de ce COALESCE en faisant simplement : `orientation_di_service_id IS NOT NULL AS is_di_service` et qu'à lafin on obtiendra toujours `TRUE` ou `FALSE` (mais pas `NULL`)

J'ai loupé un truc ?

Reply:
tu as raison ! bien vu - trop fan de coalesce j'ai un sorte de toc avec cette fonction 🤣